### PR TITLE
Create `FeedbackAnalysisResult` business logic

### PR DESCRIPTION
**Pull request by botirjonabdumannonovv**

### CODE REVIEW

File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Infrastructure/FeedbackAnalysisResults/Mappers/FeedbackAnalysisResultMapper.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿using AutoMapper;
+using Feedback.Analyzer.Application.FeedbackAnalysisResults.Models;
+using Feedback.Analyzer.Domain.Entities;
+
+namespace Feedback.Analyzer.Infrastructure.FeedbackAnalysisResults.Mappers;
+
+/// <summary>
+/// Mapper class for mapping between <see cref="FeedbackAnalysisResult"/> and <see cref="FeedbackAnalysisResultDto"/>.
+/// </summary>
```
Comment (sulton-max) [inline src/Feedback.Analyzer.Backend/Feedback.Analyzer.Infrastructure/FeedbackAnalysisResults/Mappers/FeedbackAnalysisResultMapper.cs:9]: Remove, don't add docs for mappers and validators

File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Persistence/Repositories/FeedbackAnalysisResultRepository.cs [hunk 2]
```diff
@@ -24,16 +24,4 @@ public new IQueryable<FeedbackAnalysisResult> Get(Expression<Func<FeedbackAnalys
...
-        CancellationToken cancellationToken = default)
-    {
-        return base.CreateAsync(feedbackAnalysisResult, commandOptions, cancellationToken);
-    }
-
-    public new ValueTask<FeedbackAnalysisResult?> DeleteByIdAsync(Guid feedbackId, CommandOptions commandOptions = default,
-        CancellationToken cancellationToken = default)
-    {
-        return base.DeleteByIdAsync(feedbackId, commandOptions, cancellationToken);
-    }
```
Comment (sulton-max) [inline src/Feedback.Analyzer.Backend/Feedback.Analyzer.Persistence/Repositories/FeedbackAnalysisResultRepository.cs:38]: Why did you remove Create method ? 

Feedback analysis result won't be updated or deleted, but we should definitely be able to create one ))

File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Application/FeedbackAnalysisResults/Services/IFeedbackAnalysisResultService.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+   
+    /// <summary>
+    /// Asynchronously retrieves a feedback analysis result by its ID.
+    /// </summary>
+    /// <param name="feedbackId">The ID of the feedback analysis result to retrieve.</param>
+    /// <param name="queryOptions">Options for querying.</param>
+    /// <param name="cancellationToken">A cancellation token to cancel the asynchronous operation.</param>
+    /// <returns>A <see cref="ValueTask{TResult}"/> representing the asynchronous operation, containing the feedback analysis result, or null if not found.</returns>
+    ValueTask<FeedbackAnalysisResult?> GetByIdAsync(Guid feedbackId, QueryOptions queryOptions = default, CancellationToken cancellationToken = default);
+}
```
Comment (sulton-max) [inline src/Feedback.Analyzer.Backend/Feedback.Analyzer.Application/FeedbackAnalysisResults/Services/IFeedbackAnalysisResultService.cs:39]: Add Create method as well

**Review by sulton-max — CHANGES_REQUESTED** [review]

**Review by sulton-max — DISMISSED** [review]
Good job 👍

**Review by rashidovasadbek — APPROVED** [review]

**Review by isxoqovxasanboy — APPROVED** [review]

**Review by sulton-max — APPROVED** [review]

**Pull request merged**